OpenRA.Mods.D2k/Traits/AttractsWorms.cs

```diff
@@ -20,10 +20,10 @@ public class AttractsWormsInfo : UpgradableTraitInfo
 		public readonly int Intensity = 0;
 
 		[Desc("Noise percentage at Range step away from the actor.")]
-		public readonly int[] Falloff = { 100, 100, 25, 11, 6, 4, 3, 2, 1, 0 };
+		public readonly int[] Falloff = { 100, 100, 25, 12, 6, 3, 2, 1, 0 };
```

Comment:
At this point is it safe to change default values (which mods may rely on), instead of changing our yaml?